pkg/config/virtio.go

```diff
@@ -371,7 +378,7 @@ func (dev *VirtioGPU) FromOptions(options []option) error {
 		dev.Height = defaultVirtioGPUResolutionHeight
 	}
 
-	return dev.validate()
+	return dev.Validate()
```

Comment:
This looks like the right way to validate - after creating from options. Maybe we don't need a public Validate() but add the missing validation in VirtioBlk?